### ISSUE TITLE
Installation trouble

### ISSUE THREAD

**Issue by Flandan** [bayangan1991/PYXInput#1]
Hey, I have been trying to get this to install for the past hour or so. The pip installation goes through with no complaints but when trying to import it gives the error:
`ModuleNotFoundError: No module named 'PYXInput'`

At first I thought this might be because I had python 3.5 but after uninstalling and reinstalling under python 3.6, I still cannot get it to find the package.

I'm not having any luck troubleshooting this myself. Any ideas?

Edit: Interestingly when I reinstall with it already installed (using --upgrade) I get this:
```
Installing collected packages: PYXInput
  Found existing installation: PYXInput 0.1
    Can't uninstall 'PYXInput'. No files were found to uninstall.
  Running setup.py install for PYXInput ... done
Successfully installed PYXInput-0.1
```

So it seems like no files are being copied. Maybe a problem with the setup.py?

Edit2: Another issue I had was the installation of ScpVBus-x64. The version you have included fails to install with this issue appearing in the logs:
```
The hash for the file is not present in the specified catalog file. The file is likely corrupt or the victim of tampering.
```

However installing directly from the link on the vJoy site works fine.

**Comment by bayangan1991** [comment]
Hey @Flandan, working on it now. I haven't made many packages for python

**Comment by Flandan** [comment]
@bayangan1991 no Worries, me neither.

**Comment by bayangan1991** [comment]
Hey @Flandan, I have discovered a couple of issues. You are correct in saying that the setup.py was incorrect. I have fixed that so the package now installs properly.

Other issues are due to the fact that I recompiled my DLL in debug mode, meaning the dependancies are all screwed up. I have opened another issue tracker for that. If you have Visual Studio 10+ installed is might still work though.

It is late here in Sydney though so I will fix that tomorrow.
